src/main/docker/kafdrop.sh

```diff
@@ -64,4 +64,4 @@ ARGS="--add-opens=java.base/sun.nio.ch=ALL-UNNAMED -Xss256K \
      $HEAP_ARGS \
      $JVM_OPTS"
 
-exec java $ARGS -jar /kafdrop*/kafdrop*jar ${CMD_ARGS}
+exec java $ARGS -Dloader.path=/extra-classes -jar /kafdrop*/kafdrop*jar ${CMD_ARGS}
```

Comment:
If the dir is not mounted is not a problem right?
I mean what happen if dir doesn't exist?

Reply:
That's not an issue (tested it). It just doesn't load extra classes.